common/test/acceptance/tests/studio/test_studio_course.py

```diff
@@ -0,0 +1,436 @@
+
+        # In the real workflow you will be at the dashboard page
+        # after you log in. This test was intermittently failing on the
+        # first (asset) page load with a 404.
+        # Not exactly sure why, so adding in a visit
+        # to the dashboard page here to replicate the usual flow.
+        self.dashboard_page = DashboardPage(self.browser)  # pylint:disable=attribute-defined-outside-init
+        self.dashboard_page.visit()
+
+        # Verify that each page is available
+        for page in self.pages:
+            page.visit()
+
+
+@attr('shard_2')
+class CourseTeamPageTest(StudioCourseTest):
+    """ As a course author, I want to be able to add others to my team """
+    def _make_user(self, username):
+        """ Registers user and returns user representation dictionary as expected by `log_in` function """
+        user = {
+            'username': username,
+            'email': username + "@example.com",
+            'password': username + '123'
+        }
+        AutoAuthPage(
+            self.browser, no_login=True,
+            username=user.get('username'), email=user.get('email'), password=user.get('password')
+        ).visit()
+        return user
+
+    def setUp(self, is_staff=False):
+        """
+        Install a course with no content using a fixture.
+        """
+        super(CourseTeamPageTest, self).setUp(is_staff)
+
+        self.other_user = self._make_user('other')  # pylint:disable=attribute-defined-outside-init
+        self.dashboard_page = DashboardPage(self.browser)  # pylint:disable=attribute-defined-outside-init
+        self.page = CourseTeamPage(  # pylint:disable=attribute-defined-outside-init
+            self.browser, self.course_info['org'], self.course_info['number'], self.course_info['run']
+        )
+        self._go_to_course_team_page()
+
+    def _go_to_course_team_page(self):
+        """ Opens Course Team page """
+        self.page.visit()
+        self.page.wait_until_ready()
+
+    def _expect_refresh(self):
+        """
+        Wait for the page to reload.
+        """
+        self.page = CourseTeamPage(
+            self.browser, self.course_info['org'], self.course_info['number'], self.course_info['run']
+        )
+        self._go_to_course_team_page()
+
+    def _assert_current_course(self, visible=True):
+        """ Checks if current course is accessible to current user """
+        self.dashboard_page.visit()
+        courses = self.dashboard_page.list_courses()
+
+        def check_course_equality(course1, course2):
+            """ Compares to course dictionaries using org, number and run as keys"""
+            return (
+                course1['org'] == course2['org'] and
+                course1['number'] == course2['number'] and
+                course1['run'] == course2['run']
+            )
+
+        actual_visible = any((check_course_equality(course, self.course_info) for course in courses))
+
+        self.assertTrue(actual_visible == visible)
```

Comment:
Nit: can use assertEqual(actual, expected) which produces a somewhat better error message if it fails.